internal/server/unlink_person.go

```diff
@@ -0,0 +1,67 @@
+package server
+
+import (
+	"github.com/ministryofjustice/opg-go-common/template"
+	"github.com/ministryofjustice/opg-sirius-lpa-frontend/internal/sirius"
+	"net/http"
+	"strconv"
+)
+
+type UnlinkPersonClient interface {
+	Person(sirius.Context, int) (sirius.Person, error)
+	UnlinkPerson(sirius.Context, int, int) error
+}
+
+type unlinkPersonData struct {
+	XSRFToken string
+	Success   bool
+	Error     sirius.ValidationError
+
+	Entity sirius.Person
```

Comment:
maybe call this `Person` since `Entity` elsewhere is used to populate a little context of what case/person you're looking at (and is hopefully a temporary solution before the full page UI is sorted out)